token-handler/Cargo.toml

```diff
@@ -17,6 +17,12 @@ path = "../common/transaction"
 [dependencies.bls-signature]
 path = "../common/bls-signature/"
 
+[dependencies.header-verifier]
+path = "../header-verifier/"
+
+[dependencies.tx-batch-module]
+path = "../common/tx-batch-module"
```

Comment:
I don't think we use this module anymore `tx-batch-module`

Reply:
we do, we use it in `emit_transfer_failed_events` function 
`let tx_nonce = self.get_and_save_next_tx_id();`

Reply:
The module is deprecated, later you can do another PR to delete it and move the nonce thing somewhere else